src/unit_tests/os_crypto/sha1/test_sha1_op.c

```diff
@@ -13,9 +13,6 @@
 #include <cmocka.h>
 
 #include "../headers/shared.h"
-#include "../../os_crypto/sha1/sha1_op.h"
-#include "../../wrappers/libc/stdio_wrappers.h"
-#include "../../wrappers/common.h"
```

Comment:
Is necessary this modification?

Reply:
Cleaning unnecessary headers improves compilation times.

Reply:
Ditto

Reply:
Done